doc/architecture/sensor_view.adoc

```diff
@@ -0,0 +1,8 @@
+= Sensor view
+
+Sensor view imitates the input of sensors.
```

Comment:
```suggestion
Sensor-view imitates the input of sensors.
```
@pmai since "sensors" have a special meaning in OSI, "input" may be correct here though I would add ....input of sensor models. 
Confusion may arrise in the sense that sensor-view is what a sensors sees and thus one can expect a definition to be something like:
Sensor-view imitates the **output** of sensors.

Reply:
CCB Meeting 04.08.2021:
- Sensor view provides the input to OSI sensor models.